src/components/RegistrationList.tsx

```diff
@@ -223,6 +226,13 @@ export const RegistrationListItemContent = ({
           )}
         </Box>
       )}
+      {onRemoveRelated && (
+        <DeleteIconButton
+          onClick={onRemoveRelated}
+          tooltip={t('registration.resource_type.research_data.remove_relation')}
+          data-testid={dataTestId.registrationWizard.resourceType.removeRelationButton(registration?.identifier ?? '')}
```

Comment:
Hvis man kunne fjerne relasjon uten å hatt en registration identifier hadde vi nok hatt et problem 😅 
```suggestion
          data-testid={dataTestId.registrationWizard.resourceType.removeRelationButton(registration.identifier)}
```